src/python/qtdemo.py

```diff
@@ -0,0 +1,119 @@
+import sys
```

Comment:
Consider adding a short comment at the top of this file what this script does and a usage example.